rebar.config

```diff
@@ -151,12 +151,12 @@
  [
   %% This plugin dowloads zips from github and caches them
   %% This plugin is totally optional and everything would work without it (just slower)
-  {rebar_faster_deps, {git, "https://github.com/arcusfelis/rebar3-faster-deps-plugin.git",
-      {ref, "425c75af169f40b9c1c9e471b941175c4530374f"}}},
```

Comment:
Could you give the exact reason for this? Andmaybe we could just delete it instead of commenting out if it is no longer needed?

Reply:
Yes, we could delete it.